pkg/tsdb/azuremonitor/metrics/url-builder.go

```diff
@@ -7,33 +7,50 @@ import (
 
 // urlBuilder builds the URL for calling the Azure Monitor API
 type urlBuilder struct {
+	ResourceURI string
+
+	// Following fields are deprecated and are not included in new queries.
```

Comment:
```suggestion
	// Following fields will be deprecated in grafana 9 and will not included in new queries.
```

Reply:
this has unresolved itself :( sorry I was trying to be helpful and fancy with github's suggestions feature but you've accidentally undone your comment changes now